Index: src/drivers/scsi/qedi/qedi_version.h
===================================================================
--- src.orig/drivers/scsi/qedi/qedi_version.h	2018-08-20 07:07:12.441632776 +0200
+++ src/drivers/scsi/qedi/qedi_version.h	2018-08-20 07:09:13.825473361 +0200
@@ -7,7 +7,7 @@
  * this source tree.
  */
 
-#define QEDI_MODULE_VERSION	"8.33.0.20"
+#define QEDI_MODULE_VERSION	"8.33.0.20_dup7.6"
 #define QEDI_DRIVER_MAJOR_VER		8
 #define QEDI_DRIVER_MINOR_VER		33
 #define QEDI_DRIVER_REV_VER		0
