Index: src/drivers/scsi/qedi/qedi_version.h
===================================================================
--- src.orig/drivers/scsi/qedi/qedi_version.h	2017-12-05 18:28:57.837517188 +0100
+++ src/drivers/scsi/qedi/qedi_version.h	2017-12-05 18:29:51.540787358 +0100
@@ -7,7 +7,7 @@
  * this source tree.
  */
 
-#define QEDI_MODULE_VERSION	"8.10.4.0"
+#define QEDI_MODULE_VERSION	"8.10.4.0_dup7.4"
 #define QEDI_DRIVER_MAJOR_VER		8
 #define QEDI_DRIVER_MINOR_VER		10
 #define QEDI_DRIVER_REV_VER		4
